src/chains/details/optimism.ts

```diff
@@ -0,0 +1,44 @@
+import { nativeCurrencyTemplate } from '../../common/consts/tokens'
+import { ChainInfo } from '../types'
+import { RAW_CHAINS_FILES_PATH } from '../const'
+
+const optimismLogo = `${RAW_CHAINS_FILES_PATH}/images/optimism-logo.svg`
+
+// See https://github.com/wevm/viem/blob/main/src/chains/definitions/optimism.ts
+export const optimism: ChainInfo = {
+  id: 10,
+  label: 'Optimism',
+  logo: { light: optimismLogo, dark: optimismLogo },
+  nativeCurrency: {
+    ...nativeCurrencyTemplate,
+    chainId: 10,
+    logoUrl:
+      'https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images/1/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2/logo.png',
```

Comment:
Is it different from mainnet, arb1 and base?

Reply:
no, I don't need to override there. fixed